build/optaplanner-build-parent/pom.xml

```diff
@@ -820,6 +820,39 @@
         <sonar.pullrequest.base>${env.ghprbTargetBranch}</sonar.pullrequest.base>
       </properties>
     </profile>
+    <profile>
+      <id>android-api-compatibility-check</id>
```

Comment:
Let's move it to the `fullProfile` to see what needs to get fixed before merging.

Reply:
Fixed